data/scripts.ts

```diff
@@ -1048,7 +1050,7 @@ export const Scripts: BattleScriptsData = {
 		if (!moveData.secondaries) return;
 		for (const target of targets) {
 			if (target === false) continue;
-			const secondaries: SecondaryEffect[] =
+			const secondaries: Dex.SecondaryEffect[] =
```

Comment:
? Shouldn’t this be `this.dex`?

Reply:
No, it's a type.

Reply:
It does not even seem possible to define a type in `this.dex` :(

Reply:
oh im dumb

also, rip